### ISSUE TITLE
Initial and Current value column names are confusing

### ISSUE THREAD

**Issue by DerekSansom** [a85/POSTMan-Chrome-Extension#10697]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues

### Describe the Issue

When duplicating an environment all the recent (last few weeks) variables are missing from the copy
And also when exporting an environment the latest variables are missing in the resulting json file

### Steps To Reproduce

1. Click on "Environment Quick Look"
2. Click on "Edit"
3. add an environment variable
4. wait a day or so
5. repeat steps 1 and 2 then click on "Cancel" to get the "Manage Environments" dialogue
6. Copy the edited environment
7. inspect the copy, the latest variables are missing.

### Screenshots or Videos
![Original_env](https://user-images.githubusercontent.com/6429523/155558698-7929565d-e387-4b03-a3c8-1208e753649c.png)

![Copy of environment](https://user-images.githubusercontent.com/6429523/155556228-c7284a0f-a667-4bf2-b3f3-abd3ed157c6c.png)

### Environment Information

```markdown
- Operating System: Ubuntu 20.04.4
- Platform Type: Laptop
- Postman Version: v7.36.5
```

**Comment by malvikach** [comment]
@DerekSansom thanks for writing to us. Happy to help you with this. 

Let me know if you are storing the values in the current value column, if yes- these values are session values and are not stored on postman's database. If you open a new session, the current values would be lost. 
Read more about it [here](https://learning.postman.com/docs/sending-requests/managing-environments/)

However, I see that only your recent variables are getting impacted. Would you be able to share a few more details with me like your environment JSON (remove sensitive values)

I would also request you to upgrade to our latest postman version and give the above issue a try there.
Let me know if you still face it.

**Comment by DerekSansom** [comment]
Hi Malvika,
Yes I edit "current value", or programatically set it, I don't seem able to
edit "initial value"

Is it possible to download or save in some way these current values,
and maybe columns should be renamed to reflect session or non-session,

Derek

**Comment by malvikach** [comment]
@DerekSansom yes, you can make use of _'persist all'_ to copy all current values to initial values. Similarly, you can make use of _'Reset All_' to copy the initial values to the current value column. 

The app will save the values when you persist them in initial values column. 

Regarding renaming the columns, I agree that it might be slightly confusing. We have taken note of this. I am updating your issue heading to reflect the same. 

Let us know if you face any other problems. Happy to get on a call and help you out!

**Comment by skatikia** [comment]
@DerekSansom It took us a while, but here we are- announcing a new variables interface that makes 'initial' and 'current' values in terms of the past! 🎉

With the new variables table, you always see ONE VALUE for a variable- the value that's actually going to be passed when you use the variable, with an option to share the value with your teammates or share it to the cloud to use Postman's cloud-based functionalities, such as monitors (the reason we had 'initial' values). You can start using the new interface with Postman **V11.64.0** and read all about this update here: https://blog.postman.com/engineering/a-simpler-and-local-first-variables-experience/

<img width="1189" height="697" alt="Image" src="https://github.com/user-attachments/assets/203672f8-e095-4d95-a7cc-cbc65a412a65" />

We hope you like the updated variables interface as much as we do 😊
